### PR TITLE
Migrate to 0x instant

### PR DESCRIPTION
**Pull request by justinbarry**

### CODE REVIEW

File: src/modules/account/containers/account-deposit.js [hunk 1]
```diff
@@ -1,11 +1,36 @@
...
+      const assetData = assetDataUtils.encodeERC20AssetData(repTokenAddress);
+
+      // eslint-disable-next-line
+      zeroExInstant.render(
+        {
+          availableAssetDatas: [],
+          defaultSelectedAssetData: assetData,
+          orderSource: "https://api.radarrelay.com/0x/v2"
+        },
+        "body"
```
Comment (fragosti) [inline src/modules/account/containers/account-deposit.js]: this should probably be "#app"

File: src/modules/account/components/account-deposit/account-deposit.jsx [hunk 2]
```diff
@@ -141,9 +61,11 @@ export default class AccountDeposit extends Component {
         </div>
         <div className={Styles.AccountDeposit__main}>
           <div className={Styles.AccountDeposit__description}>
-            {shapeShiftConverter}
-            <div />
-            {airSwapConverter}
+            <div className={Styles.AccountDeposit__0xInstantButton}>
+              <button onClick={openZeroExInstant}>
+                Buy REP using 0x instant.
```
Comment (fragosti) [inline src/modules/account/components/account-deposit/account-deposit.jsx]: Just making sure "instant" is capitalized. Is that done via CSS?

Comment (justinbarry) [reply src/modules/account/components/account-deposit/account-deposit.jsx]: Yes.

**Comment by grahamperich** [comment]
I'm curious about why the AirSwap widget was removed here. Should we not allow both of these decentralized atomic swap solutions (0x and AirSwap) and let the user choose whichever method they prefer? 

Would you accept a PR that puts the AirSwap widget back into the UI?

**Comment by bthaile** [comment]
Yes, we would accept a PR for adding back AirSwap.

**Pull request merged**